builder/ebsvolume/step_snapshot_ebs_volumes.go

```diff
@@ -55,11 +56,14 @@ func (s *stepSnapshotEBSVolumes) Run(ctx context.Context, state multistep.StateB
 					ResourceType: aws.String("snapshot"),
 					Tags:         tags,
 				}
-
+				description := configVolumeMapping.SnapshotDescription
+				if description == "" {
+					description = fmt.Sprintf("Packer: %s", time.Now().String())
```

Comment:
Same here, I would suggest leaving the description empty by default.

Reply:
Thanks for the comment.
The fix is pushed.